lib/packwerk/run_context.rb

```diff
@@ -14,15 +14,17 @@
 require "packwerk/package_set"
 require "packwerk/privacy_checker"
 require "packwerk/reference_extractor"
+require "packwerk/node_processor_factory"
```

Comment:
I assume this `require` belongs in a later commit, since this file doesn’t exist (yet).